### PR TITLE
[TACHYON-633] Rename dataserver messages RPCBlock* to RPCBlockRead*

### PR DESCRIPTION
**Pull request by ooq**
@gpang

### CODE REVIEW

File: common/src/main/java/tachyon/network/protocol/RPCBlockWriteResponse.java [hunk 1]
```diff
@@ -43,7 +43,8 @@ public RPCBlockWriteResponse(long userId, long blockId, long offset, long length
    * Creates a {@link RPCBlockWriteResponse} object that indicates an error for the given
    * {@link RPCBlockWriteRequest}.
    *
-   * @param request The {@link RPCBlockWriteRequest} to generated the {@link RPCBlockResponse} for.
+   * @param request The {@link RPCBlockWriteRequest} to generated
+   * the {@link RPCBlockReadResponse} for.
```
Comment (gpang) [inline common/src/main/java/tachyon/network/protocol/RPCBlockWriteResponse.java]: Is this indented the correct amount?

File: servers/src/main/java/tachyon/worker/netty/DataServerHandler.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
     ctx.close();
   }
 
-  private void handleBlockRequest(final ChannelHandlerContext ctx, final RPCBlockRequest req)
+  private void handleBlockRequest(final ChannelHandlerContext ctx, final RPCBlockReadRequest req)
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/netty/DataServerHandler.java]: Maybe this should be "handleBlockReadRequest"

**Comment by ooq** [comment]
@gpang Thanks for the comments. Fixed.

**Comment by gpang** [comment]
LGTM.

Thanks for the change!

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**